generators/client/templates/angular/src/main/webapp/app/layouts/profiles/_page-ribbon.component.ts

```diff
@@ -39,7 +39,7 @@ export class PageRibbonComponent implements OnInit {
     constructor(private profileService: ProfileService) {}
 
     ngOnInit() {
-        this.profileService.getProfileInfo().subscribe((profileInfo) => {
+        this.profileService.getProfileInfo().then((profileInfo) => {
```

Comment:
another option to avoid this call would be to pass the `profileInfo` to the component as a property from the navbar component